specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/WebApps.json

```diff
@@ -21567,6 +21567,415 @@
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots/{slot}/deployWorkflowArtifacts": {
+      "post": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "Creates the artifacts for web site, or a deployment slot.",
+        "description": "Description for Creates the artifacts for web site, or a deployment slot.",
+        "operationId": "WebApps_DeployWorkflowArtifactsSlot",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/siteNameParameter"
+          },
+          {
+            "name": "slot",
+            "in": "path",
+            "description": "Name of the deployment slot.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Artifacts deployed."
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Deploys workflow artifacts": {
+            "$ref": "./examples/PostDeployWorkflowArtifacts.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots/{slot}/workflows": {
+      "get": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "List the workflows for a web site, or a deployment slot.",
+        "operationId": "WebApps_ListInstanceWorkflowsSlot",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/siteNameParameter"
+          },
+          {
+            "name": "slot",
+            "in": "path",
+            "description": "Name of the deployment slot.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Workflow information returned.",
+            "schema": {
+              "$ref": "#/definitions/WorkflowEnvelopeCollection"
+            }
+          },
+          "404": {
+            "description": "Workflow with an ID of {workflowName} is not running.",
+            "x-ms-error-response": true
+          },
```

Comment:
This is a list API .. There is no resource called {workflowname}

Reply:
Correct response is an empty array

Reply:
Removed 404 response type. This was copied in error from the original definition in Microsoft.Logic, but there is no special 404 behavior for this API.